src/components/card-list/card-list.component.jsx

```diff
@@ -5,17 +5,17 @@ import Card from '../card/card.component';
 import BackToTop from '../back-to-top/back-to-top.component';
 
 const CardList = ({ pokemon }) => (
-  <div>
-    <div className="card-list">
+  <>
+    <section className="card-list">
       {
-        pokemon.map((pokemon, index) => (
```

Comment:
using a looping index as a key in React is not safe or consistent. `id` exists in the object which is used for key instead.

Reply:
updated to use name as key